src/resources/resources.service.spec.ts

```diff
@@ -24,4 +117,156 @@ describe("ResourcesService", () => {
+                title: expect.any(String),
+                teamMemberId: expect.any(Number),
+                createdAt: expect.any(Date),
+                updatedAt: expect.any(Date),
+            });
+
+            expect(prismaMock.voyageTeam.findUnique).toHaveBeenCalledWith({
+                where: { id: mockTeamId },
+            });
+            expect(prismaMock.teamResource.findFirst).toHaveBeenCalledWith({
+                where: {
+                    url: dtoCreateMock.url,
+                    addedBy: {
+                        voyageTeam: {
+                            id: mockTeamId,
+                        },
+                    },
+                },
+            });
+            expect(prismaMock.teamResource.create).toHaveBeenCalledWith({
+                data: { ...dtoCreateMock, teamMemberId: mockTeamMemberId },
+            });
+        });
+    });
+
+    describe("findAllResources", () => {
+        it("should find all resources", async () => {
+            prismaMock.voyageTeam.findUnique.mockResolvedValue(mockVoyageTeam);
+            prismaMock.teamResource.findMany.mockResolvedValue(mockResourceArr);
+
+            const result = await service.findAllResources(
+                requestMock,
+                mockTeamId,
+            );
+            expect(result[0]).toEqual({
+                id: expect.any(Number),
+                url: expect.any(String),
+                title: expect.any(String),
+                teamMemberId: expect.any(Number),
+                addedBy: {
+                    member: {
+                        firstName: expect.any(String),
+                        lastName: expect.any(String),
+                        id: expect.any(String),
+                        avatar: expect.any(String),
+                    },
+                },
+                createdAt: expect.any(Date),
+                updatedAt: expect.any(Date),
+            });
+
+            expect(prismaMock.voyageTeam.findUnique).toHaveBeenCalledWith({
+                where: { id: mockTeamId },
+            });
+            expect(prismaMock.teamResource.findMany).toHaveBeenCalledWith({
+                where: {
+                    addedBy: {
+                        voyageTeam: {
+                            id: mockTeamId,
+                        },
+                    },
+                },
+                include: {
+                    addedBy: {
+                        select: {
+                            member: {
+                                select: {
+                                    firstName: true,
+                                    lastName: true,
+                                    id: true,
+                                    avatar: true,
+                                },
+                            },
+                        },
+                    },
+                },
+                orderBy: { createdAt: "desc" },
+            });
+        });
+    });
```

Comment:
add a test for error cases
```suggestion
        it("should throw error if voyage team not found", async () => {
            prismaMock.voyageTeam.findUnique.mockResolvedValue(null);

            await expect(service.findAllResources(requestMock, mockTeamId)).rejects.toThrow();
        });
    });
```